packages/gatsby-theme-carbon/src/components/Aside/Aside.module.scss

```diff
@@ -3,6 +3,7 @@
   position: relative;
   padding-top: $spacing-04;
   top: -$spacing-04;
+  width: 84%;
```

Comment:
I don’t really remember where this value came from,  can we handle this with padding so we don’t need to target individual elements (like the p’s below). Sometimes I think it’s helpful to assume that the component isn’t being used with mdx. Can we guarantee the internal content will be a paragraph?

Reply:
This value came from Jeanne when sitting down next to me, to get design correct. I can move it to padding.